### PR TITLE
Replacing the wrong 'Shared Catalogs' link to correct one

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

According to the description, the link "Shared Catalogs" should open the topic page "Share Catalogs", but it opens a page for a general description of B2B functions.
This pull request (PR) replaces the wrong link to correct one on the Customer Groups topic page 

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/customers/customer-groups.html

### CODE REVIEW

File: src/customers/customer-groups.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Customer Groups
 
 Customer groups determine which discounts are available and the tax class that is associated with the group. The default customer groups are General, Not Logged In, and Wholesale.
 
-- {:.b2b-only}The selection of customer groups includes all regular customer groups, and shared catalogs, even if [Shared Catalogs]({% link stores/b2b-features.md %}) is not enabled in the configuration. Only one customer group or shared catalog can be assigned to a [company]({% link customers/account-companies.md %}) at a time.
+- {:.b2b-only}The selection of customer groups includes all regular customer groups, and shared catalogs, even if [Shared Catalogs]({% link catalog/catalog-shared.md %}) is not enabled in the configuration. Only one customer group or shared catalog can be assigned to a [company]({% link customers/account-companies.md %}) at a time.
```
Comment (hguthrie) [inline src/customers/customer-groups.md:7]: I see why you want this changed. It is confusing. 

After some research, I think the intention is to draw your attention to enabling the Shared Catalog feature, which is in step 3 of the Configure B2B features topic. What do you think?

Perhaps:

```suggestion
- {:.b2b-only}The selection of customer groups includes all regular customer groups and shared catalogs, even if **Shared Catalogs** is not enabled in the configuration. See [Configure B2B features]({% link stores/b2b-features.md %}) to enable shared catalogs. Only one customer group or shared catalog can be assigned to a [company]({% link customers/account-companies.md %}) at a time.
```

**Review by hguthrie — CHANGES_REQUESTED** [review]
Yes, this is confusing. I have an idea...

**Comment by hguthrie** [comment]
@EvheniiaKolkhiieva You're really killing' it, by the way. Thanks for all the great edits!

File: src/customers/customer-groups.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/customer-groups.md:7]: Hi @hguthrie fun fact, on the Shared Catalogs page there is also a link that leads to the page of all B2B functions, so the customer should get there anyway (I hope😁)
image
But I completely trust your experience, and I like your version. I will implement this with the next commit
Many thanks ☺️

**Comment by EvheniiaKolkhiieva** [comment]
@hguthrie Thanks for all the time you take to check.
This is awesome ☀️

File: src/customers/customer-groups.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/customers/customer-groups.md]: Done @hguthrie, I think looks great, thank you!

Could you please check and approve?

**Review by hguthrie — APPROVED** [review]

**Comment by hguthrie** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@hguthrie the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1243`.

**Pull request merged**